### ISSUE TITLE
Manual Balances Table should treat price as already in user selected currency.

### ISSUE THREAD

**Issue by kelsos** [rotki/rotki#8669]
## Problem Definition

`usdPrice` property is already converted in the user selected currency but the `AmountInput` component in the ManualBalancesTable treats it as a value in `USD` and displays it after performing a double conversion of the value.

## TODO

Change the `<AmountInput/>` that displays the usdPrice in the `ManualbalancesTable.vue` so that it does not perform a conversion and shows the value as it is, since it's already in the target currecy

### TLDR

Removing the fiat-currency="USD" should display the proper value:

https://github.com/rotki/rotki/blob/e986ea1cebee2a91c634a68d4ccace7073d26855/frontend/app/src/components/accounts/manual-balances/ManualBalanceTable.vue#L235

To confirm compare with dashboard as it should show the proper value
